package.json

```diff
@@ -80,7 +80,6 @@
     "validate.js": "0.12.0"
   },
   "napa": {
-    "bootstrap-sidebar": "https://github.com/asyraf9/bootstrap-sidebar/archive/v0.2.2.tar.gz",
```

Comment:
![2DV](https://user-images.githubusercontent.com/278968/70306159-f6c9d600-17ba-11ea-9082-dd8f1705bb10.gif)

Reply:
our biggest fans :)

Reply:
These are the guys i'm always sitting next to on 12 hour plane rides!